src/dapps/aave-dapp/components/AaveAmountForm.vue

```diff
@@ -142,11 +139,18 @@ export default {
   computed: {
     hasAmount() {
       return BigNumber(this.amount).gt(0);
+    },
+    checkIfNumerical() {
+      const regex = new RegExp('^-?[0-9]+.?[0-9]*$');
```

Comment:
This regex would accept values like 123-123 or 234234x2

If we only need commas and decimals, perhaps we could use: `^-?[0-9]+[,.]?[0-9]*$`

Reply:
ooh got it

Reply:
not sure if this regex works its erroring out for me but i am trying ^(0|[1-9][0-9]*)$

Reply:
Yeah I think it's the linter.  Could surpress it or use this `^-?[0-9]+[.]?[0-9]*$` to single out the period character

Reply:
okay ill try it!